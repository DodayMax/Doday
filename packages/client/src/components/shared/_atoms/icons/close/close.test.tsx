import * as React from 'react';
import { shallow } from 'enzyme';
import { Icons } from '@shared';

describe('Close', () => {
  it('renders correctly without props', () => {
    const icon = shallow(<Icons.Close />);

    expect(icon).toMatchSnapshot();
  });
});
