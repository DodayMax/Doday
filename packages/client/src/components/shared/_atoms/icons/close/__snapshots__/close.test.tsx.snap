// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Close renders correctly without props 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <g
        fill="none"
        fillRule="evenodd"
      >
        <path
          d="M0 0h24v24H0z"
          fill="none"
        />
        <path
          d="M13.045 12.2l6.35-6.475a.75.75 0 0 0-1.07-1.05l-6.34 6.465-6.6-6.475a.75.75 0 1 0-1.05 1.07l6.59 6.465-6.59 6.465a.75.75 0 0 0 1.05 1.07l6.6-6.475 6.34 6.465a.75.75 0 0 0 1.07-1.05l-6.35-6.475z"
          fill="black"
        />
      </g>,
      "height": 20,
      "viewBox": "0 0 24 24",
      "width": 20,
      "xmlns": "http://www.w3.org/2000/svg",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <path
            d="M0 0h24v24H0z"
            fill="none"
          />,
          <path
            d="M13.045 12.2l6.35-6.475a.75.75 0 0 0-1.07-1.05l-6.34 6.465-6.6-6.475a.75.75 0 1 0-1.05 1.07l6.59 6.465-6.59 6.465a.75.75 0 0 0 1.05 1.07l6.6-6.475 6.34 6.465a.75.75 0 0 0 1.07-1.05l-6.35-6.475z"
            fill="black"
          />,
        ],
        "fill": "none",
        "fillRule": "evenodd",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "d": "M0 0h24v24H0z",
            "fill": "none",
          },
          "ref": null,
          "rendered": null,
          "type": "path",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "d": "M13.045 12.2l6.35-6.475a.75.75 0 0 0-1.07-1.05l-6.34 6.465-6.6-6.475a.75.75 0 1 0-1.05 1.07l6.59 6.465-6.59 6.465a.75.75 0 0 0 1.05 1.07l6.6-6.475 6.34 6.465a.75.75 0 0 0 1.07-1.05l-6.35-6.475z",
            "fill": "black",
          },
          "ref": null,
          "rendered": null,
          "type": "path",
        },
      ],
      "type": "g",
    },
    "type": "svg",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <g
          fill="none"
          fillRule="evenodd"
        >
          <path
            d="M0 0h24v24H0z"
            fill="none"
          />
          <path
            d="M13.045 12.2l6.35-6.475a.75.75 0 0 0-1.07-1.05l-6.34 6.465-6.6-6.475a.75.75 0 1 0-1.05 1.07l6.59 6.465-6.59 6.465a.75.75 0 0 0 1.05 1.07l6.6-6.475 6.34 6.465a.75.75 0 0 0 1.07-1.05l-6.35-6.475z"
            fill="black"
          />
        </g>,
        "height": 20,
        "viewBox": "0 0 24 24",
        "width": 20,
        "xmlns": "http://www.w3.org/2000/svg",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <path
              d="M0 0h24v24H0z"
              fill="none"
            />,
            <path
              d="M13.045 12.2l6.35-6.475a.75.75 0 0 0-1.07-1.05l-6.34 6.465-6.6-6.475a.75.75 0 1 0-1.05 1.07l6.59 6.465-6.59 6.465a.75.75 0 0 0 1.05 1.07l6.6-6.475 6.34 6.465a.75.75 0 0 0 1.07-1.05l-6.35-6.475z"
              fill="black"
            />,
          ],
          "fill": "none",
          "fillRule": "evenodd",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "d": "M0 0h24v24H0z",
              "fill": "none",
            },
            "ref": null,
            "rendered": null,
            "type": "path",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "d": "M13.045 12.2l6.35-6.475a.75.75 0 0 0-1.07-1.05l-6.34 6.465-6.6-6.475a.75.75 0 1 0-1.05 1.07l6.59 6.465-6.59 6.465a.75.75 0 0 0 1.05 1.07l6.6-6.475 6.34 6.465a.75.75 0 0 0 1.07-1.05l-6.35-6.475z",
              "fill": "black",
            },
            "ref": null,
            "rendered": null,
            "type": "path",
          },
        ],
        "type": "g",
      },
      "type": "svg",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    "disableLifecycleMethods": true,
  },
}
`;
